src/modules/Core/Wallets/api.js

```diff
@@ -45,6 +56,26 @@ export const getReceiveAddress = (wallet: EdgeCurrencyWallet, currencyCode: stri
   return wallet.getReceiveAddress ? wallet.getReceiveAddress({ currencyCode }) : Promise.resolve(dummyEdgeReceiveAddress)
 }
 
+export const makeSpendInfo = (paymentProtocolInfo: EdgePaymentProtocolInfo): Promise<EdgeSpendInfo> => {
+  const { domain, memo, merchant, spendTargets } = paymentProtocolInfo
+
+  const name = domain === BITPAY.domain ? BITPAY.merchantName(memo) : merchant || domain
+  const notes = memo
+
+  return Promise.resolve({
+    networkFeeOption: 'high',
```

Comment:
why is the `networkFeeOption` set to `high` here?

Reply:
I copied the example
https://developer.airbitz.co/javascript/#getpaymentprotocolinfo